.env

```diff
@@ -5,7 +5,17 @@ SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
 SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
 SIMORGH_CONFIG_URL=https://config.test.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.test.bbc.com/ws/includes
+SIMORGH_INCLUDES_BASE_AMP_URL=https://news.test.files.bbci.co.uk
```

Comment:
Eek, this might have been an accidental commit.. We'll need to revert this.

Reply:
oops, reverted.